### ISSUE TITLE
Missing syntax highlighting for JSDoc

### ISSUE THREAD

**Issue by brendanfalkowski** [atom/language-javascript#632]
### Description

About a month ago, Atom stopped highlighting the JSDoc comment before JS functions.

![screen shot 2018-12-11 at 3 55 42 pm](https://user-images.githubusercontent.com/214924/49838017-a792c600-fd5d-11e8-8a8f-0774da579e0c.png)

Previously the different tokens were colored to aid readability.

### Steps to Reproduce

Write a JSDoc before a function in a JS file.

**Expected behavior:** highlighter should color the comment

**Actual behavior:** comment is all gray

**Reproduces how often:** always

### Versions

Atom 1.33.0
macOS Mojave

**Comment by Aerijo** [comment]
There was a regression when Tree-sitter was enabled by default, but that has been fixed in all releases now.

Could you provide a minimum (not) working example, so I can verify it's not the context that is the problem? There seems to be lines above the screenshot you've provided.

Can you also confirm this reproduces in safe mode? We need that check, because it disables all community packages to ensure they aren't causing the issue (if they are, you'd need to raise the issue on their repo first).

**Comment by brendanfalkowski** [comment]
Restarted in safe mode, but same outcome:

![screen shot 2018-12-11 at 4 30 12 pm](https://user-images.githubusercontent.com/214924/49839093-72d53d80-fd62-11e8-9ecf-44e7cd77e6af.png)

Here's that simplified test case's code:

```javascript
/**
 * Function description here.
 *
 * @param {object} elem
 * @param {string} name
 */
function realityCheck (elem, name) {
    return true;
}
```

**Comment by borisovg** [comment]
I confirm this is still a problem - #600 was supposed to fix it but clearly something didn't work out.

**Comment by rsese** [comment]
Hmm, looks like it didn't make it into 1.33.0 but it's fixed in 1.34.0-beta0.  Just to clarify, does anyone see this problem on the beta?

**Comment by rsese** [comment]
Thanks @Aerijo - going to go ahead and close then since we close issues that are fixed but just not released.
